_posts/2020-08-20-review-apps-post.markdown

```diff
@@ -0,0 +1,308 @@
+Slack channel notifying of their plan to deploy a particular service, cc'ing the
+engineers that are involved in other PRs that are part of the deploy. In the example
+above, an engineer on Team B would notify relevant stakeholders of Team A, giving
+that team the opportunity to flag if their work is not yet safe to deploy.
+
+While these 2 strategies are impactful:
+
+- Semi-unstructured communication is prone to breakdown: the notified engineers
+  on Team A might be pairing or in a meeting and Team B deploys anyways.
+- Without true continuous delivery model, it's a challenge to operationalize
+  very frequent production deploys. Moreover, the problem compounds as the team
+  grows and the velocity of work increases.
+
+If only there was a way to allow Team A and B to work without risking stepping
+on each other toes!
+
+We'll discuss how review apps provide this safety, but first another related
+problem.
+
+## Problem 2.0: Complex Work Needs Many Eyes
+
+But before it gets better, it's going to get a bit worse.
+
+While working on any mature distributed system is a challenge, some work is
+particularly challenging or risky. Generally, risk is reduced when many
+skilled eyes can review the risky thing.
+
+This class of work might include changes to authorization flows, page
+redesigns or infrastructural upgrades (i.e. a NodeJS upgrade).
+
+For example, to feel safe deploying [a major version upgrade of Artsy's design
+system in Force][styled-systems-upgrade-pr] the most pragmatic way forward was
+to deploy that PR to a server where other engineers could collaborate on QA.
+
+*If a single shared staging environment is the only non-production server to
+deploy to, the chances that work lands on staging in an unsafe state is high*. While
+staging being unsafe isn't _itself_ a bad thing, many bad things can result:
+
+1. [Bad] Blocked Deploys
+
+	If staging is unsafe and this dangerous state is discovered, then top priority
+is getting a service back into a safe state. While working to get a service
+back to a healthy state, new features can't be delivered.
+
+	In aggregate, this can be a sizeable productivity loss.
+
+2. [Worse] Unsafe Deploys
+
+	If staging in unsafe and this dangerous state is not discovered before a production
+deploy (for example, the unsafe code might be from another team, as described above),
+then end-users might interact with a service that just isn't working. No good.
+
+3. [Terrible] Fear
+
+	> Fear is the mind-killer.
+	[Dune][dune-fear-quote]
+
+	Alright, a bit over the top, but the risk of unsafe or blocked deploys can
+implicitly or explicitly result in teams shying away from complex work.
+
+	This avoidable fear might result in increased technical debt or not taking on
+certain projects.
+
+	It's generally bad for business and does not lead to a pleasant work environment!
+
+## Problem Recap & Review App Introduction
+
+In review, when working on complex tasks, particularly on shared services, it is
```

Comment:
I find this use of "review" confusing: my immediate thought was that it must relate to "review apps". Maybe you could say "To recap" (the repetition from the subtitle is not too bad IMHO)

Reply:
```suggestion
To recap, when working on complex tasks, particularly on shared services, it is
```